### DISCUSSION TITLE
Multiple fields uniqueness validation

### DISCUSSION THREAD

**Discussion by vol1ura** [jeremyevans/sequel#2121] [General]
After `validates_unique([:column1, :column2])` validation the error key will contain `[:column1, :column2]`, which can cause some problems, because the key have to be a symbol, for example, `sentry-raven` (old version) crashes on such keys.

It is possible to correct this behavior in Sequel? Perhaps, can I do it like in Rails, where one field is validated anyway, just in the context of other fields and the error is assigned to the selected field.

**Comment by jeremyevans** [comment]
You want to use the error_splitter plugin: https://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/ErrorSplitter.html

**Reply by vol1ura** [reply]
Thanx!!!
